spec/support/stubs/sidekiq/api.rb

```diff
@@ -1,4 +1,4 @@
-class Sidekiq
+module Sidekiq
```

Comment:
Change sidekiq class to module, since the actual Sidekiq constant is a module too.